operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -1043,6 +1043,9 @@ message BaseConfig {
   // URL to use for validating webhook.
   string validationURL = 2;
 
+  // For istioctl usage to disable istio config crds in base
+  google.protobuf.BoolValue enableIstioConfigCRDs = 3;
```

Comment:
I think this is ok - although typically throughout the code base, where possible, we use exclusive logic (i.e. if enableIstioConfigCRDs { .. } - not if !disableIstioConfigCRDs { ... }.

Happy to approve if the logic is reversed (that way the default, of false, does nothing). Also I am not clear why the word Config is needed.

Cheers,
-steve

Reply:
Generally it's considered a best practice to test positive conditionals, so `if xEnabled` rather than `if !xDisabled`. Using negatives to represent a positive condition leads to double negatives which gets confusing to read and understand.

I'm in favor of leaving it as-is. And to be honest, we should make that a best practice for Istio APIs going forward to the extend possible.

Reply:
Tend to agree with Nate

Reply:
At the very least its more common for us - `prometheus.enabled` for example is all over our api

Reply:
ya I guess Pilot doesn't do this all that much unless its a feature flag, in which case, it follows this pattern.

Cheers
-steve

Reply:
In general yes, but at the same time there are advantages to having the default case match a negative conditional, due to Go's zero-values. Really we want `indent non-default flow` so something like `if disable { return/continue/break }`